project/dagger-query-ui.sh

```diff
@@ -12,5 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+xdg-open src/com/google/daggerqueryui/index.html
+start src/com/google/daggerqueryui/index.html
```

Comment:
Well windows can't run `*.sh` scripts anyways so I don't think the `start` variant has value here. You might want to make available a separate `*.bat` script for windows if you really want :stuck_out_tongue:

Reply:
O, makes sense. Let's just remove `start` command and mention in README, that Windows users need to run server manually (they still can open bash scripts and see, how to do it).